contrib/storage-hive/core/pom.xml

```diff
@@ -142,11 +142,6 @@
         </exclusion>
       </exclusions>
     </dependency>
-    <dependency>
```

Comment:
To follow excluding MapR specific code from the default profile, please move `format-maprdb` to `mapr` profile in the contrib/pom.xml

```
  <modules>
    <module>storage-hbase</module>
    <module>storage-hive</module>
    <module>storage-mongo</module>
    <module>storage-jdbc</module>
    <module>storage-kafka</module>
    <module>storage-kudu</module>
    <module>storage-opentsdb</module>
    <module>data</module>
    <module>udfs</module>
  </modules>
```

```
    <profile>
      <id>mapr</id>
      <modules>
        <module>format-maprdb</module>
      </modules>
    </profile>
```